lib/internal/child_process.js

```diff
@@ -230,12 +230,12 @@ util.inherits(ChildProcess, EventEmitter);
 
 
 function flushStdio(subprocess) {
-  if (subprocess.stdio == null) return;
-  subprocess.stdio.forEach(function(stream, fd, stdio) {
-    if (!stream || !stream.readable || stream._readableState.readableListening)
-      return;
-    stream.resume();
-  });
+  if (subprocess.stdio === null || subprocess.stdio === undefined) return;
+  for (let i = 0; i < subprocess.stdio.length; i++) {
```

Comment:
@notarseniy `make lint` should have reported that issue.  `make lint` is part of `make test` so if you ran tests, that should have come up as a lint error after the tests all passed. If the linter didn't report it, then there is probably a bug in our custom lint rule.